samples/commands/execute_command/README.md

```diff
@@ -0,0 +1,529 @@
+# Commands Execution
+
+[**Return to main sample list**](../../README.md)
+
+This is a semi-interactive sample that allows you to use the AWS IoT [Commands](https://docs.aws.amazon.com/iot/latest/developerguide/iot-remote-command.html)
+service to receive and process remote instructions.
+
+> [!NOTE]
+> The sample walkthrough assumes the [AWS CLI](https://aws.amazon.com/cli/) has been installed and configured in order
+> to invoke control plane operations that are not possible with the device SDK.
+
+In a real use case, control plane commands (the AWS CLI commands) would be issued by applications under control of
+the customer, while the data plane operations (the actions performed by the sample application) would be issued by
+software running on the IoT device itself.
+
+Using the IoT Commands service and this sample requires an understanding of two closely-related but different service terms:
+* **AWS IoT Command** - metadata describing a task that the user would like one or more devices to run.
+* **AWS IoT Command Execution** - metadata describing the state of a single device's attempt to execute an AWS IoT Command.
+
+In particular, you can define an **AWS IoT Command** and then send it multiple times to the same device. The device will
+try to execute each received **AWS IoT Command Execution**.
+
+Another important feature of AWS IoT command service is using different MQTT topics for different payload formats. This
+means that your device can choose to receive only AWS IoT commands with specified payload format.
+AWS IoT Commands service distinguishes the following payload formats:
+- JSON
+- CBOR
+- generic (i.e. everything else)
+
+This also means that if your device wants to receive JSON **and** CBOR payloads, it needs to subscribe to both topics
```

Comment:
"This also means that if your device wants to receive JSON **and** CBOR payloads, it needs to subscribe to both topics using two separate API calls." 
->
"If your device wants to receive both JSON and CBOR payloads, it will need to subscribe to both topics using two separate API calls."